tests/cancun/eip4788_beacon_root/test_beacon_root_contract.py

```diff
@@ -14,10 +12,6 @@
     - post
     - valid_call
 
-    The following arguments *need* to be parametrized or the test will not be generated:
-
-    -
-
```

Comment:
Hey @spencer-tb, I don't think any arguments need to be parametrized for these tests, so I cleaned it out. Could you double check plz? :slightly_smiling_face: Compare with the comment in `test_excess_blob_gas.py`, for example.